appveyor.yml

```diff
@@ -115,7 +106,7 @@ after_build:
   - py.test tests\unit
 
   # Python integration tests
-  - py.test tests\integration
+  # - py.test tests\integration
```

Comment:
We will be running integration tests in `nupic.regression`. I don't think there is any need to run them here.